### ISSUE TITLE
Question regarding the aff_map in PCAA module

### ISSUE THREAD

**Issue by paolopertino** [lsa1997/PCAA#4]
Hello, thanks for sharing this great work.

I have a question regarding the computation of the aff_map in the PCAA module. I see in your implementation [here](https://github.com/lsa1997/PCAA/blob/f281b0716b1cf9279b48e0d314ec077a677f6160/networks/caanet.py#L74) that a scale attribute is defined but never used. Should we multiply it [at line 105 of the `caanet.py`](https://github.com/lsa1997/PCAA/blob/f281b0716b1cf9279b48e0d314ec077a677f6160/networks/caanet.py#L105) file before applying the softmax?

That is, instead of doing:
```python
...
aff_map = torch.matmul(query, key.transpose(2, 3)) # [B, bin_num_h * bin_num_w, rH * rW, K]
aff_map = F.softmax(aff_map, dim=-1)
...
```
should we do this way?
```python
...
aff_map = self.scale * torch.matmul(query, key.transpose(2, 3)) # [B, bin_num_h * bin_num_w, rH * rW, K]
aff_map = F.softmax(aff_map, dim=-1)
...
```

Thanks a lot!
